src/main/res/values/strings.xml

```diff
@@ -115,10 +115,10 @@
     <!-- Menu Prefs -->
 
     <string name="pref_menus_post_context_items_key" translatable="false">pref_menus_post_context_items_2</string>
-    <string name="pref_menus_post_context_items_title">Post context menu items</string>
+    <string name="pref_menus_post_context_items_title">Pop-up menu items</string>
```

Comment:
Agreed. I went with "long-press" instead of "long-click" since "press" is input-method agnostic, but I will happily change it if you prefer.
The preference is now under a new **Posts** PreferenceCategory which makes it clear by context that the preference is for posts.